.changeset/nice-peas-marry.md

```diff
@@ -0,0 +1,10 @@
+---
+"@hopper-ui/svg-icons": patch
+---
+
+Add color variables for custom rich-icons styling
+
+to change the background color, use the variable `--hop-richicon-placeholder-background`
```

Comment:
we usually capitalize the component name like so : `--hop-Button-primary-color`

Reply:
I based myself on the structure of `@hopper-ui/icons` -> `--hop-richicon-placeholder-icon-color` 
should change around as we introduce a major version for icons. 

@alexasselin008, what do you think?

Reply:
you're right, this is a mistake on my part. --hop-RichIcon would indeed be more standard